simple-wp-helpdesk/admin/class-reporting.php

```
@@ -0,0 +1,241 @@
+/**
+ * Returns ticket counts grouped by status.
+ *
+ * @since 3.0.0
+ * @return array<string, int> Map of status label => count.
+ */
+function swh_report_status_breakdown() {
+	$statuses = swh_get_statuses();
+	$result   = array();
+	foreach ( $statuses as $status ) {
+		// phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_query
+		$count             = (int) ( new WP_Query(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '68', column '36'). (undefined)

(MissingImport)